src/styles/color.ts

```diff
@@ -0,0 +1,8 @@
+export const COLOR = {
+    RED: 'D32D2F',
+    BLACK: '111111',
+    GREY_1: '767676',
+    GREY_2: 'CDCDCD',
+    GREY_3: 'EFEFEF',
```

Comment:
GRAY로 바꿔주세요!

그리고 숫자가 붙는 이런 상수는 100단위 이상으로 해주면 좋아요! 숫자 사이의 값이 새로 생길 수 있어서 그런데 예를 들어 프로젝트를 진행하다 디자이너가 gray1.5라는 컬러를 추가할 수도 있어요 지금 실제로 추가적으로 뷰가 나올 상황이기도 하니까 수정하면 좋을 것 같아요!

```suggestion
    GRAY_100: '767676',
    GRAY_200: 'CDCDCD',
    GRAY_300: 'EFEFEF',
```